### ISSUE TITLE
Creds aren't loaded. Are you logged in? 

### ISSUE THREAD

**Issue by GeraldSingh** [mxrch/GHunt#414]
I have been able to use a google cloud terminal to install and load GHunt. However, when I try to use it to login (with the GHunt companion aiding with the cookies) I don't get a message saying that creds are stored anywhere. And when I try to look up an email I get the following message:

[-] No stored cookies found

[-] Creds aren't loaded. Are you logged in ?

I'm new to this so unsure if I'm missing something. Can this run with the Google Cloud terminal?

**Comment by sealldeveloper** [comment]
Are you using the server sync method as that would not work over Google Cloud, you would need to copy the cookies.

Otherwise I haven't used Google Cloud terminal, just the first thing that comes to mind.

**Comment by StevenGolfGS** [comment]
Yeah I was trying with the companion app. I'll try with manually inputting the cookies

**Comment by mxrch** [comment]
@StevenGolfGS Did you managed to do what you want ?

**Comment by StevenGolfGS** [comment]
@mxrch not yet but I may not have properly copied the cookies when I tried that. I will try again later this week

**Comment by mxrch** [comment]
@StevenGolfGS You can try with the Companion app using the 1st method, so you don't have to copy paste

**Comment by sealldeveloper** [comment]
> @StevenGolfGS You can try with the Companion app using the 1st method, so you don't have to copy paste

Will that work over the Google Cloud? I don't think the listener would work there...
